submissions/5mountains/tiny-js-world-oop/index.js

```diff
@@ -0,0 +1,105 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Resident {
+  constructor(species, name, gender, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+  }
+  introducing() {
```

Comment:
If you replace `introducing` with magic `toString` you won't need to call this method explicitly in string context. E.g. `print(inhabitant)` would do, `toString` will be called implicitly.

Reply:
Okey, I'll try it.